core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -285,15 +285,20 @@
      */
     public static final String FIELD_TRACING_SAMPLING_MODE_PER_AUTH_ID = "sampling-mode-per-auth-id";
 
+    /**
+     * The default regular expression validating IDs contain only legal characters.
+     */
+    public static final String DEFAULT_ID_REGEX = "[a-zA-Z0-9-_\\.]+";
```

Comment:
@sophokles73 it is indeed the same regex, just without end delimiters. I moved it here and based other ones on it. Let me know if you have any other ideas on how improve on this.